src/groovy/au/org/emii/gogoduck/worker/Worker.groovy

```diff
@@ -42,6 +45,10 @@ class Worker {
         shellCmd.call(cmdOptions)
     }
 
+    String getTempDir() {
+        return System.getProperty("java.io.tmpdir")
```

Comment:
Now it is actually `/tmp` and that's fine. gogoduck.sh will not try to remove this directory, this is the directory it'll use to create new temporary directories under.